lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3006,8 +3006,7 @@ namespace {
 
       auto templateParamTypeUsedInSignature =
           [decl](clang::TemplateTypeParmDecl *type) -> bool {
-        // TODO(SR-13809): we will want to update this to handle dependent
-        // types when those are supported.
+        // TODO: we will want to update this to handle dependent types when those are supported. (https://github.com/apple/swift/issues/56206)
```

Comment:
I'd vote for `TODO`, `FIXME`, and `MARK` mentioning issue in parentheses if that's not too much work to make it happen...

Reply:
I find it less discoverable, but I don’t mind. What about the char limit?

Reply:
I think it’s okay to go over the 80 char limit in such circumstances.